test/e2e/README.md

```diff
@@ -179,6 +179,14 @@ manifest to the control-plane Docker container:
 go test -v github.com/vmware-tanzu/antrea/test/e2e -provider=kind
 ```
 
+`generate-manifest.sh` supports generating the Antrea manifest with different
+Antrea configurations. Run `./hack/generate-manifest.sh --help` to see the
+supported config options. Note, after changing the Antrea manifest with a
+different set of Antrea config options, you might need to delete the old Antrea
+ConfigMap in the Kind cluster (that is created by the previous Antrea
+deployments with the old manifest), to make sure your new Antrea deployment will
+take the new ConfigMap with the changes.
```

Comment:
I don't think the issue is with the Antrea deployment. This issue is specifically with the test framework which doesn't look for the "right" ConfigMap. Maybe the test framework should be improved instead.

Reply:
Makes sense. I can check if you point to me some information here.

Reply:
it's this code: https://github.com/vmware-tanzu/antrea/blob/bb6cd1016e2d2863847ce13052b951e27a654b14/test/e2e/framework.go#L1539. Maybe a simple solution is to look at the CreationTimestamp and pick the most recent one? Instead of stopping the iteration at the first matching ConfigMap

Reply:
I thought we should delete the old CM?

Reply:
That's not a requirement. We don't delete it on Antrea upgrades in real-life scenarios

Reply:
Ok. But why we do not do GetAntreaConfigMap() here?

Reply:
we should :) probably the code predates `GetAntreaConfigMap`
making that change will resolve the issue